cmd/spire-server/cli/entry/util_test.go

```diff
@@ -45,3 +49,93 @@ func selectorToFlag(t *testing.T, selectors []*common.Selector) StringsFlag {
 
 	return resp
 }
+
+func TestRegisterParseFile(t *testing.T) {
+	p := path.Join(util.ProjectRoot(), "test/fixture/registration/good.json")
```

Comment:
is there any bad json we should test against?

Reply:
587ac9c
Invalid JSON is now also included in the test case.
Also refactored test functions.